installer/roles/local_docker/templates/docker-compose.yml.j2

```diff
@@ -138,6 +150,27 @@ services:
     volumes:
       - "{{ docker_compose_dir }}/redis.conf:/usr/local/etc/redis/redis.conf:ro"
       - "{{ docker_compose_dir }}/redis_socket:/var/run/redis/:rw"
+      - "{{ docker_compose_dir }}/memcached_socket:/var/run/memcached/:rw"
```

Comment:
This should be dropped when rebasing too.

Reply:
fixed up